ferretdb/ferretdb.go

```diff
@@ -147,6 +147,10 @@ func New(config *Config) (*FerretDB, error) {
 // Run runs FerretDB until ctx is canceled.
 //
 // When this method returns, listener and all connections, as well as handler are closed.
+//
+// It is required to run this method in order to initialise the listeners with their respective
+// IP address and port. Calling methods which require the listener's address (eg: `FerretDB.MongoDBURI()`
```

Comment:
Go uses `[that syntax]` for links. I think `[MongoDBURI]` should work. But please check it with `task godocs`

Reply:
sure.. let me check that

Reply:
done